macOS builds broken on Boost stage

Starting from https://github.com/NilFoundation/zkllvm/commit/ba21e4f4b30b809a9b436baa22f958fe1ed471b6 the macOS build is broken. Need to fix it. 

```
-- Could NOT find Boost (missing: Boost_INCLUDE_DIR) 
CMake Error at /usr/local/Cellar/cmake/3.26.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Boost (missing: Boost_INCLUDE_DIR container)
-- Configuring incomplete, errors occurred!
```

--- a/.github/workflows/build_macos.yml
+++ b/.github/workflows/build_macos.yml
@@ -62,7 +62,7 @@ jobs:
 
       - name: Configure CMake
         env:
-          BOOST_ROOT: "/usr/local/Cellar/boost@1.76/1.76.0_3"
+          BOOST_ROOT: "/usr/local/opt/boost@1.76"
           LDFLAGS: "-L/usr/local/opt/icu4c/lib"
           CPPFLAGS: "-I/usr/local/opt/icu4c/include"
         run: |

